cmd/influx/replication.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/influxdata/influx-cli/v2/pkg/cli/middleware"
+	"github.com/urfave/cli"
+)
+
+func newReplicationCmd() cli.Command {
+	return cli.Command{
+		Name:   "replication",
+		Usage:  "Replication stream management commands",
+		Hidden: true, // Remove this line when all subcommands are completed
+		Subcommands: []cli.Command{
+			newReplicationCreateCmd(),
+			newReplicationDeleteCmd(),
+			newReplicationListCmd(),
+			newReplicationMetricsCmd(),
+			newReplicationUpdateCmd(),
+		},
+	}
+}
+
+func newReplicationCreateCmd() cli.Command {
+	return cli.Command{
+		Name:   "create",
+		Usage:  "Create a new replication stream",
+		Before: middleware.WithBeforeFns(withCli(), withApi(true), middleware.NoArgs),
+		Flags: append(
+			commonFlags(),
+		),
+		Action: func(ctx *cli.Context) {
+			fmt.Println("replication create command was called")
+		},
+	}
+}
+
+func newReplicationDeleteCmd() cli.Command {
+	return cli.Command{
+		Name:   "delete",
+		Usage:  "Delete an existing replication stream",
+		Before: middleware.WithBeforeFns(withCli(), withApi(true), middleware.NoArgs),
+		Flags: append(
+			commonFlags(),
+		),
+		Action: func(ctx *cli.Context) {
+			fmt.Println("replication delete command was called")
+		},
+	}
+}
+
+func newReplicationListCmd() cli.Command {
+	return cli.Command{
+		Name:    "list",
+		Usage:   "List all replication streams",
+		Aliases: []string{"find", "ls"},
+		Before:  middleware.WithBeforeFns(withCli(), withApi(true), middleware.NoArgs),
+		Flags: append(
+			commonFlags(),
+		),
+		Action: func(ctx *cli.Context) {
+			fmt.Println("replication list command was called")
+		},
+	}
+}
+
+func newReplicationMetricsCmd() cli.Command {
+	return cli.Command{
+		Name:   "metrics",
```

Comment:
Hmmm as I understand it, the only queue info we'll have for each stream is:
* Current # of bytes in the queue
* Max # of bytes in the queue

I'm not sure we need a separate command to show that vs. showing it in the output of `list`. What do you think?

Reply:
I think it's fine to combine if that's all the info we need to show - I'll go ahead and change to reflect this.